hadoop-client-modules/hadoop-client/pom.xml

```diff
@@ -290,6 +302,10 @@
           <groupId>com.github.pjfanning</groupId>
           <artifactId>jersey-json</artifactId>
         </exclusion>
+        <exclusion>
```

Comment:
I'm no expert on maven but I would have expected that by excluding com.github.pjfanning:jersey-json above that its jettison transitive dependency would have been excluded as a result. I could be wrong - I can have a look at what happens with the hadoop-common 3.3.5 later.

Reply:
I added jettison as direct dependency of hadoop-common and (if i did not mistaking) these are pulling in hadoop-common in them. Just being cautious, they excluded the jersey-json (and jettison with it), so I excluded jettison too (which would come from hadoop-common now)

Reply:
I think this is too cautious. If the excluded are not needed they should be removed. At least in theory, the explicit include of jettison should be enough.

Reply:
thx for the quick review! appreciating :)